test/test_dmclock_server.cc

```diff
@@ -16,6 +16,46 @@
 #include "dmclock_util.h"
 #include "gtest/gtest.h"
 
+#include <dmtest-config.h>
+#ifdef HAVE_SYS_PRCTL_H
+#include <iostream>
+#include <sys/prctl.h>
+#include "common/errno.h"
```

Comment:
common/errno.h is specific to the ceph build tree. dmclock needs to build independently (i.e., "git clone git@github.com:ceph/dmclock.git" and then build). I tried removing this #include and it seemed to build and run tests and the simulation. The build system had /usr/include/sys/prctl.h and according to CMakeCache.txt "HAVE_SYS_PRCTL_H:INTERNAL=1". So is that #include necessary?